docs/roadmap.md

```diff
@@ -0,0 +1,44 @@
+
+## Goals
+
+Looking at wasmCloud development in the longer-term, we strive to achieve the following goals:
+
+1. **Leverage as many WebAssembly standards as possible (WASI, Wasm components, wasi-cloud, etc.)**
+1. Provide a seamless developer experience for building, testing, and deploying WebAssembly components
+1. Be the best way to build vendor-less components for WebAssembly applications
+1. Bring joy to distributed computing
+1. Leverage cloud-native standards where possible (OAM, OCI, CloudEvents, etc.)
+
+wasmCloud began in 2019 with the goal of securely connecting capabilities to applications, leveraging WebAssembly to provide a language and platform-agnostic unit of compute. This loose coupling of capabilities and applications is at the core of wasmCloud's ability to bring joy to distributed computing. This allows application developers to: maintain their business logic and non-functional requirements separately, build without the intricacies of platform-specific dependencies, and seamlessly distribute applications without changing their code.
+
+## wasmCloud Q3/Q4 2023 Roadmap
+
+The first goal above is the focus of our current (as of 01 August 2023) efforts. Inspired by the Bytecode Alliance blog [WebAssembly: An Updated Roadmap for Developers](https://bytecodealliance.org/articles/webassembly-the-updated-roadmap-for-developers), we're working to adopt as many WebAssembly standards as possible.
+
+1. **Leverage [WebAssembly interface types (wit)](https://github.com/WebAssembly/component-model/blob/main/design/mvp/WIT.md) for our interfaces:** we can use and contribute to the wide variety of projects that provide language support for WebAssembly components, extending wasmCloud's language support from Rust and TinyGo to include C, [Javascript](https://github.com/bytecodealliance/jco), [Python](https://github.com/dicej/componentize-py), and more as they become available.
+1. **Implement and use [wasi-cloud](https://github.com/WebAssembly/wasi-cloud-core) instead of proprietary wasmCloud interfaces:** letting users bring their own components to wasmCloud instead of requiring them to be built with wasmCloud's SDK.
+1. **Transition feature focus to Rust:** Recently accepted as [ADR-0013](https://github.com/wasmCloud/wasmCloud/blob/main/adr/0013-transition-feature-focus-to-rust.md), we aim to focus our efforts on the wasmCloud Rust runtime project to rapidly conform to standards and offer a portable, stable, high-performance wasmCloud runtime.
```

Comment:
Nit: can we title this one something different? To newcomers I'm afraid it means wasmCloud as a whole is only focusing on Rust, vs our primary runtime being written in Rust